btk/draw_blends.py

```diff
@@ -22,15 +22,13 @@ def get_center_in_pixels(blend_catalog, stamp_size, pixel_scale):
 
     Args:
         blend_catalog: Catalog with entries corresponding to one blend.
-        stamp_size: In arcseconds.
-        pixel_scale: Number of pixels per arcsecond.
-
+        wcs: astropy.wcs.WCS object corresponding to the image
     Returns:
         `astropy.table.Column`: x and y coordinates of object centroid
     """
-    center = (stamp_size / pixel_scale - 1) / 2
-    dx = blend_catalog["ra"] / pixel_scale + center
-    dy = blend_catalog["dec"] / pixel_scale + center
+    dx, dy = wcs.all_world2pix(
+        blend_catalog["ra"] / 3600, blend_catalog["dec"] / 3600, 0
+    )
```

Comment:
question which is not clear to me: now that we have WCS might it be easier to not convert 'ra' and 'dec' to arcseconds to begin with? (this is done in the sampling functions). Do you see other obstacles Thomas?

Reply:
I think the core of it was to have the pixel scale and stamp size in arcseconds, which are more adapted to the weak lensing case as the galaxy studied have a size of a few arcseconds at most, which is why I kept as much arcseconds as possible (also there might be some galsim features which require arcseconds ?). The ideal case would be to have astropy wcs support arcsecs but that is a bit out of the scope of btk.

Reply:
Got it, thanks. I agree that probably better to still do the conversion of arcseconds at the beginning then.